src/Qt-RSA/qrsaencryption.h

```diff
@@ -53,7 +53,8 @@ To ensure reliable protection, it is recommended to use an exponent size of at l
         /// Fast but not stable. (using by default)
         Auto = 0,
         /// Stable but slow. (using for sig and check sig messages)
-        OneByte = 1
+        OneByte = 1,
+        Auto2 = 2
```

Comment:
This is bad name of mode. Please add a readable description for this mode and rename it.